helpers/3p/utils/index.js

```diff
@@ -373,14 +373,14 @@ utils.context = function (thisArg, locals, options) {
  * @api public
  */
 
-utils.options = function (thisArg, locals, options) {
+utils.options = function (thisArg, locals) {
```

Comment:
This function isn't a helper, and is only used [here](https://github.com/bigcommerce/paper-handlebars/blob/260f8d1e67ecb6e9c38ef468c0a1e48fc81642f4/helpers/option.js#L24) where the helper implementation ensures the correct object is passed in. It probably doesn't need this change.